sig-api-machinery/annual-report-2023.md

```diff
@@ -14,17 +14,22 @@
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
-<!--
-   Note: This list is generated from the KEP metadata in kubernetes/enhancements repository.
-      If you find any discrepancy in the generated list here, please check the KEP metadata.
-      Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
--->
+Yes, in general SIG API Machinery owns a large portion of the Kubernetes codebase. We are always looking to welcome more contributors that help up improve and keep healthy this core important part of Kubernetes.
+Just to mention a few areas:
```

Comment:
Could you also please link to some issues/documents that can help a prospective contributor to get started?